### DISCUSSION TITLE
Which Robots support GoToLocationCapability?

### DISCUSSION THREAD

**Discussion by digodigo** [Hypfer/Valetudo#1852] [Q&A / Support] [answered]
Hello,

I'd like to implement an automation which sends my Dreame L10 Pro (running Valetudo 2023.05.0) to the kitchen every now and then so it sits there and waits to be emptied. I found the `GoToLocationCapability` but that seems not to be supported by Dreame. So I'd like to ask two questions:

1. Does anyone know where I can find an overview as to which robot supports the `GoToLocationCapability` in its firmware?
2. Is there an idea I didn't come up with that implements the wished behaviour by other means?

I tried to find answers up and down the net, yet to no avail. Any insight would be greatly appreciated :)

**Comment by rimrul** [comment, marked as answer]
> 1. Does anyone know where I can find an overview as to which robot supports the `GoToLocationCapability` in its firmware?

https://valetudo.cloud/pages/usage/implementation-overview.html#Overview

**Reply by digodigo** [reply]
D'oh! Thanks a bunch! 
I have no idea how I have missed that as it is just the next page after the description of capabilities themselves.
